platform/android/java/app/build.gradle

```diff
@@ -75,6 +75,11 @@ android {
     }
 
     defaultConfig {
+        // The default ignore pattern for the 'assets' directory includes hidden files and directories which are used by Godot projects.
+        aaptOptions {
+            ignoreAssetsPattern "!.svn:!.git:!.ds_store:!*.scc:<dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"
```

Comment:
`picasa.ini`, that's nostalgic :)

I'm not sure to understand the change though, does it make those patterns non ignored (due to the `!`)?
If so, why are files like `.ds_store` (isn't it `.DS_Store` usually BTW?) or `thumbs.db` relevant for us?

Reply:
The default pattern, defined [here](https://android.googlesource.com/platform/frameworks/base/+/b41af58f49d371cedf041443d20a1893f7f6c840/tools/aapt/AaptAssets.cpp#61), is `"!.svn:!.git:!.ds_store:!*.scc:.*:<dir>_*:!CVS:!thumbs.db:!picasa.ini:!*~"`.

That pattern includes `.*` which I've removed in the change.
The `!` is used to [avoid printing a warning](https://android.googlesource.com/platform/frameworks/base/+/b41af58f49d371cedf041443d20a1893f7f6c840/tools/aapt/AaptAssets.cpp#69).